citadel/indico_citadel/search.py

```diff
@@ -38,7 +38,7 @@ def search(self, query, access, object_type=SearchTarget.event, page=1, params=N
         # https://cern-search.docs.cern.ch/usage/operations/#advanced-queries
         q = f'{format_query(query, placeholders)} {ranges}'
         search_params = {'page': page, 'size': self.RESULTS_PER_PAGE, 'q': q, 'highlight': '_data.*',
-                         'type': object_type.name, **filter_query}
+                         'type': (x.name for x in object_types), **filter_query}
```

Comment:
Passing a generator here seems rather odd

```suggestion
                         'type': [x.name for x in object_types], **filter_query}
```